packages/itwin/ec3-widget/src/components/TemplateMenu.tsx

```diff
@@ -53,7 +53,7 @@ export const TemplateMenu = ({ template, goBack, created }: TemplateProps) => {
   const [showDeleteModal, setShowDeleteModal] = useState<boolean>(false);
   const [showReportConfirmModal, setShowReportConfirmModal] = useState<boolean>(false);
   const [selectedLabel, setSelectedLabel] = useState<EC3Label>();
-  const [selectedReport, setSelectedReport] = useState<string>();
+  const [lastReport, setLastReport] = useState<string>();
```

Comment:
What is the meaning of `lastReport`?

Reply:
If the user selects a new report and then presses cancel on the confirmation modal we need a way to get the last selected report (the combobox changes to the new report when selecting). This stores it. I thought perhaps I could put it in a simple variable but that didn't work.

Reply:
It should at least have a better name. `lastReport` sounds like it's the last report in some kind of list, not the report that user selected most recently.